src/report/errors.rs

```diff
@@ -419,3 +437,14 @@ pub fn set_show_loaded_mods(v: bool) {
 pub(crate) fn set_predicate(predicate: FilterRule) {
     Errors::get_mut().filter.predicate = predicate;
 }
+
+pub fn set_log_once_defaults() {
+    Errors::get_mut().log_once = vec![
+        // All instances of something missing are solved by adding the thing
+        ErrorKey::MissingFile,
+        ErrorKey::MissingItem,
+        ErrorKey::MissingLocalization,
+        ErrorKey::MissingPerspective,
+        ErrorKey::MissingSound,
```

Comment:
`MissingSound` is not like the others. Very few modders will modify the sound banks. Instead, if Pdx removes a sound, the modders have to find a replacement sound to use instead.